examples/process-instrumentation/process-instrumentation-grafana-dashboard-sample.json

```diff
@@ -0,0 +1,640 @@
+{
```

Comment:
could you update the readme for the library to indicate there is a example grafana dashboard here? its might be undiscovered otherwise by folks looking at the readme.

Reply:
I've updated the readme with the links; please take a look. Thank you.